src/GraphBuilder.js

```diff
@@ -35,6 +36,22 @@ export default class GraphBuilder {
     return this.dataset
   }
 
+  /**
+   * @return {string} a Turtle representation of a graph
+   */
+  async toTurtle() {
```

Comment:
If anyone can find a way to take a graph and serialize it as Turtle or JSON-LD *without* needing to add promises up the stack... I'm all ears. I've had no luck, perusing the `jsonld` library, the `rdf-ext` libraries, and the `rdfjs` libraries, and random others I've dug up via searching around desperately.